src/index.js

```diff
@@ -1,5 +1,12 @@
 import React from "react";
+import { Provider } from "react-redux";
 import ReactDOM from "react-dom";
-import App from "./App";
+import store from "./store";
```

Comment:
Is the file called `"./store"` or `"./Store"`?

Reply:
I have changed this and am now using the store as folder with relative import